modules/lkqdBidAdapter.js

```diff
@@ -165,7 +179,7 @@ function buildRequests(validBidRequests, bidderRequest) {
       bidRequests.push({
         method: 'GET',
         url: sspUrl,
-        data: sspData
+        data: Object.keys(sspData).map(function (key) { return key + '=' + sspData[key]} ).join('&') + '&'
```

Comment:
Fix lint error. `} )` should be ` })`

Also, `reduce` could be used here instead of `map` followed by `join`. See [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce)

Reply:
Fixed lint error
sspData is an Object and not an Array, so `reduce` was not applicable

Reply:
`Object.keys` returns an array. That's why you're able to use `map` and `join` currently.